libs/env/src/env.ts

```diff
@@ -522,7 +522,7 @@ const envDefinitions: Record<string, EnvDef<boolean | string | number | string[]
     desc: "limit the number of addresses in from/to of operations",
   },
   OPERATION_OPTIMISTIC_RETENTION: {
-    def: 30 * 60 * 1000,
+    def: 24 * 60 * 60 * 1000,
```

Comment:
i have a feeling that this significant change means you have a new usecase that is beyond the initial meaning of this env.

historically this was a workaround to keep the operation during the unstable phase where backend have a flawky state where it returns the optimistic tx and then it disappear again, and then appear again, etc.

because the way it's used is

```
shouldRetainPendingOperation(mainAccount, op) || // after retain logic, we need operation to appear
        !operations.some(o => o.hash === op.hash)
```

so when the op is not even appearing in the operations list, the pendingOperations will always preserve the operation.

**does this changes means that backend have a 24 hours flawky operation returning situation?**

Reply:
thanks for info. I revert it